subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -459,6 +459,14 @@ config BT_CTLR_CENTRAL_SPACING
 	  (active clock jitter) + 17040 (PDU rx) = (radio event overheads +
 	  34234) microseconds.
 
+config BT_CTLR_CONN_EVENT_LEN_UPDATE
+	bool "Update event length reservation after PHY or DLE update"
+	depends on !BT_LL_SW_LLCP_LEGACY && (BT_CTLR_DATA_LENGTH || BT_CTLR_PHY)
+	default n
```

Comment:
I think the recommendation is to not set `default n`, as it is in fact default. Also I'm wondering if the name should reflect that we're talking about slot reservation?

Reply:
You're probably right, a few issues popped up recently that will be fixed by this PR
As for the name, I am open to suggestions, or how about BT_CTLR_SLOT_RESERVATION_UPDATE